src/components/EventUserTable.js

```diff
@@ -66,7 +66,8 @@ export class EventUserTable extends Component {
         const heardFrom = {}
         response.data.forEach(user => {
           if (user.heardFromData) {
-            heardFrom[user.heardFrom] = heardFrom.hasOwnProperty(user.heardFrom) ? heardFrom[user.heardFrom] + 1 : 1
+            heardFrom[user.heardFrom] = Object.prototype.hasOwnProperty.call(heardFrom, user.heardFrom)
```

Comment:
I don't think we need `hasOwnProperty`, does this work?
```suggestion
            heardFrom[user.heardFrom] = heardFrom[user.heardFrom] ? heardFrom[user.heardFrom] + 1 : 1
```